mconfig

```diff
@@ -697,15 +698,25 @@ cat $makeit_makefile_base >> $makeit_makefile
 drawline $makeit_fragsdir/common_opts.mk
 cat $makeit_fragsdir/common_opts.mk >> $makeit_makefile
 
+drawline $makeit_fragsdir/go_common_opts.mk
+cat $makeit_fragsdir/go_common_opts.mk >> $makeit_makefile
+
+if [ -f "$makeit_fragsdir/go_${host}_opts.mk" ]; then
+	drawline $makeit_fragsdir/go_${host}_opts.mk
+	cat $makeit_fragsdir/go_${host}_opts.mk >> $makeit_makefile
+fi
 
 if [ "$appsec" = "1" ]; then
 	drawline $makeit_fragsdir/go_appsec_opts.mk
 	cat $makeit_fragsdir/go_appsec_opts.mk >> $makeit_makefile
-else
-	drawline $makeit_fragsdir/go_common_opts.mk
-	cat $makeit_fragsdir/go_common_opts.mk >> $makeit_makefile
 fi
 
+if [ "$build_runtime" = "1" ]; then
+	drawline $makeit_fragsdir/go_runtime_opts.mk
+	cat $makeit_fragsdir/go_runtime_opts.mk >> $makeit_makefile
```

Comment:
This ends up setting the build tag to ensure `singularity_runtime` components are built and defines the CGO relevant flags for the runtime C/C++ source build.